go.mod

```diff
@@ -26,19 +26,48 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	github.com/vishvananda/netlink v1.0.0
+	github.com/vishvananda/netlink v1.1.0
 	golang.org/x/net v0.0.0-20201202161906-c7110b5ffcbb // indirect
 	golang.org/x/text v0.3.4 // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.19.4
-	k8s.io/apiextensions-apiserver v0.19.2
+	k8s.io/apiextensions-apiserver v0.19.4
 	k8s.io/apimachinery v0.19.4
-	k8s.io/client-go v0.19.2
-	k8s.io/code-generator v0.19.2
-	k8s.io/component-base v0.19.2
+	k8s.io/client-go v0.19.4
+	k8s.io/code-generator v0.19.4
+	k8s.io/component-base v0.19.4
 	k8s.io/klog/v2 v2.4.0
-	k8s.io/kube-proxy v0.19.2
+	k8s.io/kube-proxy v0.19.4
+	k8s.io/kubernetes v1.19.4
```

Comment:
Since this is for master only, we really ought to go to `1.20.0-rc.0` if you can...

Reply:
`1.20.0-rc.0` hasn't landed in `openshift/kubernetes` yet afaik. I can create a card to track updating CNO deps to 1.20 when it lands?

Reply:
It looks like if I try to bump to 1.20.0-rc.0, we hit errors with `openshift/api` repo not being bumped to 1.20 yet.

```
go: finding module for package k8s.io/api/settings/v1alpha1
github.com/openshift/cluster-network-operator/pkg/controller/pki imports
        github.com/openshift/library-go/pkg/operator/certrotation imports
        github.com/openshift/library-go/pkg/operator/resource/resourceapply imports
        github.com/openshift/api imports
        k8s.io/api/settings/v1alpha1: module k8s.io/api@latest found (v0.19.4, replaced by k8s.io/api@v0.20.0-rc.0), but does not contain package k8s.io/api/settings/v1alpha1
```
I am going to create a card for myself to bump CNO go mods to 1.20.0 once the api changes land.

Reply:
Will track this as SDN-1405